FF1Lib/Classes.cs

```diff
@@ -695,18 +695,22 @@ public List<string> DoRandomizeClassNormalMode(MT19337 rng, List<string> olditem
 			}*/
 
 			// Single Spells Bonus/Malus
-			bonusNormal.AddRange(CreateSpellBonuses(rom, rng));
-			malusNormal.AddRange(CreateSpellMaluses(rom, rng));
+			// Incompatible with Spellcrafter, so only add them when Spellcrafter is not enabled
+			if (!(bool)flags.GenerateNewSpellbook)
```

Comment:
Already fixed, it works with spellcrafter now

Reply:
Ah, missed that - Reverted